app/views/api_docs/pages/release-notes.md

```diff
@@ -3,9 +3,9 @@ page_title: Register API release notes
 title: Register API release notes
 ---
 
-## v0.1 — 15 April 2024
+## v0.1 — 19 April 2024
 
-The draft version of the Register API was released on 15 April 2024. This is the first version of the API and will be subject to change.
+The draft version of the Register API was released on 19 April 2024. This is the first version of the API and will be subject to change.
```

Comment:
Yes, this could be stronger. I don't want to commit to breaking changes but have tried to make it clearer. Updated in https://github.com/DFE-Digital/register-trainee-teachers/pull/4219/commits/d9053ba5d2fdfa5011383b1aa6cc21c895074f75